server/src/config.ts

```diff
@@ -1,5 +1,22 @@
 import { Request } from "express";
 import type { Knex } from "knex";
+import { createLogger, Logger, transports, format } from "winston";
+const { combine, timestamp, splat, printf, label } = format;
+
+const level = process.env.LOG_LEVEL || "info";
```

Comment:
Ahhh, I was just trying to say reading external configuration data (i.e. determining the current logging level from `process.env.LOG_LEVEL`) should be in the `config` module, not the entirety of all the logging code!

Reply:
ah no worries! I can split it back up